tests/integration/test_client.py

```diff
@@ -0,0 +1,68 @@
+import aiohttp
+import unittest
+import uuid
+from aiohttp.test_utils import setup_test_loop, unittest_run_loop
+from os import environ as ENV
+
+from aiosparql.client import SPARQLClient
+from aiosparql.syntax import IRI
+
+
+sample_data = """\
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix ex: <http://example.org/stuff/1.0/> .
+
+<http://www.w3.org/TR/rdf-syntax-grammar>
+  dc:title "RDF/XML Syntax Specification (Revised)" ;
+  ex:editor [
+    ex:fullname "Dave Beckett";
+    ex:homePage <http://purl.org/net/dajobe/>
+  ] .
+"""
+sample_format = "text/turtle"
+
+
+class Client(unittest.TestCase):
+    @unittest_run_loop
+    async def test_crud(self):
+        await self.client.put(sample_data, format=sample_format)
+        await self.client.delete()
+        await self.client.post(sample_data, format=sample_format)
+        async with self.client.get(format="text/turtle") as res:
+            self.assertEqual(res.status, 200)
+            text = await res.text()
+            self.assertIn("@prefix", text)
+        async with await self.client.head(format="text/turtle") as res:
+            self.assertIn(res.status, (200, 501))
```

Comment:
I love doing integration test with a server that does not implement everything...